masterlist.yaml

```
@@ -10047,6 +10047,24 @@ plugins:
       - crc: 0xE8F7C46B
         util: 'TES4Edit v4.0.4'
 
+  - name: 'TolawaylAyleidCity_Quest.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/11526/' ]
+    msg:
+      - <<: *wildEdits
+        subs: [ 'Remove **00031D50**, **000034DD**, & **0001E727** from **TolawaylAyleidCity_Quest.esp**.' ]
+        condition: 'checksum("TolawaylAyleidCity_Quest.esp", C5C868BD) or checksum("TolawaylAyleidCity_Quest.esp", 7414E9CC)'
+    dirty:
+      - <<: *quickClean
+        crc: 0xC5C868BD
+        util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 19
+        udr: 18
```

QAC undeleting `00031D50` moves it back to its original cell, leaving two ITMs behind when it's removed after running QAC. This report is after running QAC once & then cleaning the wild edits.

```suggestion
        udr: 18
      - <<: *quickClean
        crc: 0x0F1B9C71
        util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
        itm: 2
```